client/post-editor/editor-ground-control/index.jsx

```diff
@@ -35,9 +37,12 @@ export default React.createClass( {
 		onPreview: React.PropTypes.func,
 		onPublish: React.PropTypes.func,
 		onSaveDraft: React.PropTypes.func,
+		onMoreInfoAboutEmailVerify: React.PropTypes.func,
 		post: React.PropTypes.object,
 		savedPost: React.PropTypes.object,
 		site: React.PropTypes.object,
+		user: React.PropTypes.object,
+		userUtils: React.PropTypes.object,
```

Comment:
Why do you need to pass `userUtils` through as a prop instead of just calling as an import within this file?